### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by lerfast**
Hi i did this:

- [ ] Setup new repository
- [ ] Install linters to the repository.
- [ ] Copy code from my partners work to HTML and CSS file.
- [ ] Run linters.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+<body>
+  <div class="container">
+    <header>
+      <h1 id="title">VL Survey Form</h1>
+      <p id="description">
+        Thank you for taking your time to improve the platform
+      </p>
+    </header>
+    <form id="survey-form">
+      <!--* Name -->
```
Comment (lerfast) [inline index.html:21]: Plsease do not leave this notes in your codebase. It can confuse beginners programmers

File: README.md [hunk 2]
```diff
@@ -1 +1,202 @@
-# review-code-work
```
Comment (lerfast) [inline README.md:1]: Please, remove this line so your code looks clean

**Review by lerfast — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and make all required changes.

Submit for a re- review when you've done.

Thanks

**Review by LionRouge1 — APPROVED** [review]
Hi @lerfast, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL]
- [ ] Please consider using emojis and/or gifs in your review, it makes your code review more human-friendly.

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**